src/cryptoadvance/specter/cli/cli_ext.py

```diff
@@ -109,6 +126,26 @@ def gen(org, ext_id, isolated_client, devicename, tmpl_fs_source, dryrun):
             "Enter the prefix:",
             type=str,
         )
+    if encrypted_userdata == None:
+        print(
+            """
+             Your data is currently unencrypted
+            """
+        )
+        encrypted_userdata = click.prompt(
+            "Should the data be encrypted (y/n)?",
+            type=bool,
+        )
+    if custom_version == None:
+        print(
+            """
+            The extension generation uses the latest version of specter. 
```

Comment:
I think a fallback should rather be not the lastest version, but the version ext_gen is running on. That can be found (in <= v2) in `from cryptoadvance.specter._version import version`.